server/middlewares/stats.js

```diff
@@ -78,6 +112,18 @@ const trackTxSubmissions = mung.json((body, req, res) => {
       const txOutAmount = parseTxBodyOutAmount(txBody)
 
       incrCountersBy(`${txSubmissionType}:sentOut`, txOutAmount)
+      logActivity('event', {
+        ec: 'Payments',
```

Comment:
I would rather pass an object with human readable keys and reassign those keys to the google analytics object inside `logActivity`. In general, you want to keep the internals of the function from the caller, since it may change at any time.